### ISSUE TITLE
hs.fs - Support for reading/writing Finder comments?

### ISSUE THREAD

**Issue by latenitefilms** [Hammerspoon/hammerspoon#1400]
This might be relevant: https://sourceforge.net/projects/osxutils/

**Comment by asmagill** [comment]
The code in osxutils is using AppleEvents (i.e. Applescript, but in a programmatic and hard to read way).

Much easier to use `hs.osascript.applescript` directly to do something like this:

~~~
> hs.inspect(table.pack(hs.osascript.applescript([[
tell application "Finder"
  set filePath to "init.lua" as posix file
  set comment of (filePath as alias) to "I'm a test comment"
end tell
]])))
{ true, "I'm a test comment", "'utxt'(\"I'm a test comment\")",
  n = 3
}
~~~

and

~~~
> inspect(table.pack(hs.osascript.applescript([[
tell application "Finder"
  set filePath to "init.lua" as posix file
  get comment of (filePath as alias)
end tell
]])))

{ true, "I'm a test comment", "'utxt'(\"I'm a test comment\")",
  n = 3
}
~~~

**Comment by latenitefilms** [comment]
Maybe worth adding Lua functions to `hs.fs` to do this via AppleScript then for ease of use and better documentation?

I'm happy to give this a crack, but won't have time for a little while - as I have a very big to-do list currently.

**Comment by asmagill** [comment]
Should be addressed in #1515, reopen if you discover outstanding issues.
